components/multimedia/lms/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2020 Rouven Weiler
+#
+
+BUILD_BITS=			32
+BUILD_STYLE=			archive
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		lms
+COMPONENT_FMRI =		service/multimedia/
+COMPONENT_VERSION =		7.9
+COMPONENT_PROJECT_URL =		https://github.org/Logitech/slimserver.git
+COMPONENT_SUMMARY =		Logitech Media Server
+COMPONENT_LICENSE =		GPLv2
+COMPONENT_LICENSE_FILE =	$(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION =	System/File System
+
+COMPONENT_SRC =			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE =		$(COMPONENT_SRC).tar.gz
+GIT_HASH =			sha256:520c0d1633d707a03e3e7d9fd741b7019592660b648f697e0ca1083053edec9f
+GIT_REPO =			https://github.com/Logitech/slimserver.git
+GIT_BRANCH =			public/7.9
```

Comment:
@chincheta0815 Can you discuss why should we use their Git repo instead of [tarball releases](http://downloads.slimdevices.com/LogitechMediaServer_v7.9.2/)?

Reply:
LMS is releasing code in an (in my opinion) uncommon way. The tarballs represent a version freeze after one year, e.g.: 

- 7.9.2 is right now end-of-development and end-of-support release. All the plugins that are existing and addons are not updated for that version too. In terms of samba this would represent the 4.9 release series.


- The 7.9.3 branch is the right now stable and bugfix release. All addons are updated/bugfixed and kept working against that version. I personally have very good experience with that kind of release. In terms of samba this is the 4.11 release series.

Speaking about the tarball releases there exists just a tarball for the 7.9.2 version. As a side effect it might be confusing for a maintainer not dealing each day with lms which one of these to use (noCPAN, CPAN, ...). For the desired version 7.9.3 there is no taball existing. The 7.9.3 version resides in the github.com branch named public/7.9. So I decided to use that branch form the repo.

In addition, the needed vendor/perl modules are only available in a repository on github. So it seemed just a good scheme to use the git repo(s). And, very important, there is not tarball for all the vendor/perl modules that is valid and working for version 7.9.2. So for the vendor/perl modules one has to use the repo.

Using the repo tarballs was something I decided not to do. The tarballs one gets by downloading them from github extract to a directory called slimserver-... So I ended up with "lms-base/slimserver/public/7.9" or "lms-vendor/slimserver-vendor/oi-userland/7.9" which would have caused lines of code in the Makefile that I simply could avoid by adding prep-git.mk and use that.

I hope I could explain my thoughts well and can stick to my code ;o)